src/servers/src/postgres/types.rs

```diff
@@ -524,6 +549,113 @@ pub(super) fn param_types_to_pg_types(
     Ok(types)
 }
 
+#[derive(Debug, Clone, Copy, Default)]
+pub struct PgInterval {
```

Comment:
Can we put this struct to a dedicated module say `types/interval.rs`? I'm afraid this module is going to be too large when more types added.

Reply:
thx, I will refactor it. ๐•ᴗ•๐